lib/request.js

```diff
@@ -462,6 +463,17 @@ module.exports = {
       .slice(offset);
   },
 
+  /**
+   * Get accept object.
+   * Lazily memoized.
+   *
+   * @return {Object}
+   * @api public
+   */
+  get accept() {
```

Comment:
maybe you should add `set accept()` to avoid breaking change.

Reply:
All right.

Reply:
I've pushed another commit to this PR.